Dockerfile

```diff
@@ -18,7 +18,7 @@ WORKDIR /go/src/github.com/kubermatic/machine-controller
 COPY . .
 RUN make all
 
-FROM alpine:3.12
+FROM alpine:3.16
```

Comment:
just saying: I'd recommend distroless images

Reply:
That'd be my personal preference as well. @xrstf WDYT? I agree with @kron4eg.

Reply:
I personally don't think the security gains / size savings of distroless are worth the hassle of losing the ability to debug a container quickly. But I also cannot remember the last time I exec'ed into a machine-controller pod, so... whatever the maintainers of the machine-controller think is right.